activationdemo/source/java/src/org/kaaproject/kaa/demo/activation/PagerProfileData.java

```diff
@@ -0,0 +1,69 @@
+/**
+ *  Copyright 2014-2016 CyberVision, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.kaaproject.kaa.demo.activation;
+
+import java.util.ArrayList;
+
+/**
+ * Class for storing test data and showing all functionality
+ */
+public class PagerProfileData {
```

Comment:
add json file with profile and configuration schemas into resources folder and also edit [ActivationdemoBuilder](https://github.com/kaaproject/sample-apps/blob/master/activationdemo/src/main/java/org/kaaproject/kaa/examples/activation/ActivationDemoBuilder.java)